images/Sauna.svg

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
nazov suboru ikon daj prosim malymi pismenami ako je to maju vsetky doterajsie ikony. inac to na*nix systemoch ani nepojde.

naviac ikona musi byt opravena aby mala biely, 50% outline, 1.5px - ako to maju vsetky ostatne ikony.